slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -421,21 +426,21 @@ class SlideToActView @JvmOverloads constructor (
             canvas.rotate(mArrowAngle, mInnerRect.centerX(), mInnerRect.centerY())
         }
         mDrawableArrow.setBounds(mInnerRect.left.toInt() + mArrowMargin,
-            mInnerRect.top.toInt() + mArrowMargin,
-            mInnerRect.right.toInt() - mArrowMargin,
-            mInnerRect.bottom.toInt() - mArrowMargin)
+                mInnerRect.top.toInt() + mArrowMargin,
+                mInnerRect.right.toInt() - mArrowMargin,
+                mInnerRect.bottom.toInt() - mArrowMargin)
         if (mDrawableArrow.bounds.left <= mDrawableArrow.bounds.right &&
-            mDrawableArrow.bounds.top <= mDrawableArrow.bounds.bottom) {
+                mDrawableArrow.bounds.top <= mDrawableArrow.bounds.bottom) {
             mDrawableArrow.draw(canvas)
```

Comment:
_Commenting here but this is valid for the entire file._

Please cleanup the Diff and make sure you include only changes that are relevant to your change (`mIconMargin` and related fields). Looks like you have a lot of formatting changes. I kindly ask you to remove those. I'll setup ktlint for this purpose in a separate PR.

Reply:
Ok, I will remove it. It seems we have different formating rules and I didn't realise.